### DISCUSSION TITLE
Vertical two thirds instead of horizontal two thirds

### DISCUSSION THREAD

**Discussion by robolson** [rxhanson/Rectangle#1258] [Q&A] [answered]
Recently I lost the ability to have Rectangle resize a window using the top two thirds of the screen. Now I only see the option for two thirds starting from the left or right edge. I don't know if this changed when I upgraded to Ventura or if it disappeared in an update of Rectangle.

How do I get the ability to resize with a vertical two thirds back?

**Comment by rxhanson** [comment]
Thanks for reporting. There have been no changes to the app that would cause this. Perhaps you were using a portrait (vertical) oriented display previously?

**Reply by robolson** [reply]
Yes. I was using a portrait oriented display previously. I still am. The monitor is a ThinkVision P24h-2L. I use two displays, the main one in landscape orientation and the second one in portrait orientation. I think something changed with the upgrade from MacOS Monterey to MacOS Ventura. I believe that Rectangle may only be receiving the orientation of the primary display from the OS.

**Comment by robolson** [comment, marked as answer]
I found the problem/solution. When I upgraded to MacOS Ventura the default behavior to show the menu bar on all displays was changed. I went into 'System Preferences' -> 'Desktop & Doc' -> 'Mission Control' section and checked "Displays have separate spaces". Now I once again have a menu bar on both screens, and when I click on Rectangle on the portrait mode screen, I see 2/3rds partitioning in the vertical direction.
